cfgov/v1/__init__.py

```diff
@@ -11,6 +11,7 @@
 def environment(**options):
     options.setdefault('extensions', []).append(CompressorExtension)
     env = sheerlike_environment(**options)
+    env.autoescape = True
```

Comment:
Okay so if it's autoescaping now then there's a ton of escaping done in the templates that is redundant code. Idk if it should be in the scope of this PR but it's something to be aware of.